components/brave_federated/brave_federated_service.cc

```diff
@@ -33,7 +34,7 @@ BraveFederatedService::BraveFederatedService(
       browser_context_path_(browser_context_path),
       url_loader_factory_(url_loader_factory) {
   InitPrefChangeRegistrar();
```

Comment:
nit: move to `Init()` or `decouple data_store_service_`, `eligibility_service_` and `operational_patterns_` into there own Init functions